File: beecrowd_problems/1_beginner/1002/solution.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8')
var lines = input.split('\n')

const PI = 3.14159
let raio = parseFloat(input)

let area = PI * Math.pow(raio, 2)

console.log('A=' + area.toFixed(4))

File: beecrowd_problems/1_beginner/1002/solution.cpp
#include <iostream>
#include <iomanip>
#include <math.h>

using namespace std;

int main()
{
    const double PI = 3.14159;
    double raio, area;

    cin >> raio;

    area = PI * pow(raio, 2.0);

    cout << "A=" << setprecision(4) << fixed << area << endl;

    return 0;
}